tools/nni_cmd/config_utils.py

```diff
@@ -119,4 +119,25 @@ def read_file(self):
                     return json.load(file)
             except ValueError:
                 return {}
-        return {} 
\ No newline at end of file
+        return {} 
+
+class HDFSConfig:
+    '''manage hdfs configuration'''
+    def __init__(self):
+        os.makedirs(NNICTL_HOME_DIR, exist_ok=True)
+        self.hdfs_config_file = os.path.join(NNICTL_HOME_DIR, '.hdfs')
+    
+    def get_config(self):
+        if os.path.exists(self.hdfs_config_file):
+            try:
+                with open(self.hdfs_config_file, 'r') as file:
+                    return json.load(file)
+            except Exception:
+                return {}
```

Comment:
the meaning of return `{}`?

Reply:
Return a None type or an empty dict if got error, changed to use `None`, more easy to understand.